src/main/java/dev/jbang/cli/App.java

```diff
@@ -45,7 +45,7 @@ public static void deleteCommandFiles(String name) {
 
 @CommandLine.Command(name = "install", description = "Install a script as a command.")
 class AppInstall extends BaseCommand {
-	private static final String jbangUrl = "https://github.com/jbangdev/jbang/releases/latest/download/jbang.zip";
+	private static final String jbangUrl = "www.jbang.dev/releases/latest/download/jbang.zip";
```

Comment:
no https:// ?

Reply:
DUnno, shouldn't it be https always nowadays?

Reply:
yes, thats what i'm asking ;) sholdn't it be "https://www.jbang../"